src/java.base/share/classes/java/lang/Double.java

```diff
@@ -878,7 +878,7 @@ public int hashCode() {
      */
     public static int hashCode(double value) {
         long bits = doubleToLongBits(value);
-        return (int)(bits ^ (bits >>> 32));
+        return Long.hashCode(bits);
```

Comment:
Imo these should be squashed to just 
```java
return Long.hashCode(doubleToLongBits(value));
```
since the local variable is no longer meaningful. Previously, they were required as they were retrieved twice in hash code calculation.

Same for other usages.

Reply:
Done!